pkg/users/local.js

```diff
@@ -81,13 +83,28 @@
     );
 
     if (path.length === 0) {
-        return <AccountsMain accountsInfo={accountsInfo} current_user={current_user_info && current_user_info.name} groups={groupsExtraInfo || []} />;
-    } else {
+        return (
+            <AccountsMain
+                accountsInfo={accountsInfo}
+                current_user={current_user_info && current_user_info.name}
+                groups={groupsExtraInfo || []}
+                isGroupsExpanded={isGroupsExpanded}
+                setIsGroupsExpanded={setIsGroupsExpanded}
+            />
+        );
+    } else if (path.length === 1) {
         return (
             <AccountDetails accounts={accountsInfo} groups={groupsExtraInfo || []} shadow={shadow || []}
                             current_user={current_user_info && current_user_info.name} user={path[0]} />
         );
-    }
+    } else if (path.length === 2 && path[0] === "group" && groupsExtraInfo.length > 0 && accountsInfo.length > 0) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230116-100807-e09cfe13-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)